Primo Normalized XML.js

```diff
@@ -468,19 +483,61 @@ var testCases = [
+		"items": [
+			{
+				"itemType": "book",
+				"title": "Zur Medienöffentlichkeit der Dritten Gewalt rechtliche Aspekte des Zugangs der Medien zur Rechtsprechung im Verfassungsstaat des Grundgesetzes",
```

Comment:
It is not in the PNX data, which is a little strange, because it is in the underlying data. However, I don't see anything we can do in the translator for this case.